modules/passkey/test/utils/webauthn.ts

```diff
@@ -345,15 +345,43 @@ export function decodeSignature(response: Pick<AuthenticatorAssertionResponse, '
   return { r, s }
 }
 
+/**
+ * Encodes the given WebAuthn signature into a string. Used for testing purposes.
+ *
+ * @param authenticatorData - The authenticator data as a Uint8Array.
+ * @param clientDataFields - The client data fields as a string.
+ * @param r - The value of r as a bigint.
+ * @param s - The value of s as a bigint.
+ * @returns The encoded string.
+ */
+export function getSignatureBytes(authenticatorData: Uint8Array, clientDataFields: string, r: bigint, s: bigint): string {
```

Comment:
ubernit:

Might be nicer to implement this as:

```suggestion
export function getSignatureBytes(signature: { authenticatorData: BytesLike, clientDataFields: string, r: BigNumberish, s: BigNumberish }): string {
```

Since you are always calling this function as:
```ts
getSignatureBytes(signature.authenticatorData, signature.clientDataFields, signature.r, signature.s)
```